src/app-view.js

```diff
@@ -72,6 +91,12 @@ class AppView extends React.Component {
         );
     }
 
+    handleKeyDown(event) {
+        if (event.keyCode === 83) {
+            this.setState(state => ({ isSidebarHidden: !state.isSidebarHidden }));
```

Comment:
can we put this.setState(state => ({ isSidebarHidden: !state.isSidebarHidden })); in separate function and call it in  other methods?